packages/core-data/src/resolvers.js

```diff
@@ -61,6 +61,16 @@ export function* getEntityRecord( kind, name, key = '' ) {
 	yield receiveEntityRecords( kind, name, record );
 }
 
+/**
+ * Requests an entity's record from the REST API.
+ */
+export const getRawEntityRecord = getEntityRecord;
```

Comment:
I haven't yet double-checked to confirm, but I suspect there may be a small issue with this, in that resolver completion is checked by _name_, not by _reference_. So there's a pretty good chance this is causing `getEntityRecord` to be called a second time, for `getRawEntityRecord`'s own resolution to be considered satisfied.

Assuming that I can confirm this, a couple thoughts I have on approaching a solution:

- The resolver behavior could detect completion by reference, so that references to another resolver as seen here could be managed internally by the data module.
- Or, we could create an ad hoc function wrapper here with conditions which check to see if resolvers have started for `getEntityRecord`, and only call if needed. Basically, recreating the same logic of the internal data resolver handling.

The second seems simpler, and simultaneously worrying in how it's "recreating" an existing behavior. However, I think it may not be too much an issue in practice, both because (a) the logic is quite simple and reuses common resolution selectors, and (b) it could be abstracted rather easily to a higher-order function common to both of these new resolvers.

I'm imagining something like:

```suggestion
export const getRawEntityRecord = ifNotResolved( getEntityRecord );
```

It may need to be slightly more complicated, since resolver selectors expect to receive the selector name as an argument. We can likely retrieve that by `Function#name`, and while in practice it's probably not a real problem, `Function#name` is not something we should come to rely on when minification is involved in the production build.

```suggestion
export const getRawEntityRecord = ifNotResolved( getEntityRecord, 'getEntityRecord' );
```

Reply:
> * Or, we could create an ad hoc function wrapper here with conditions which check to see if resolvers have started for `getEntityRecord`, and only call if needed. Basically, recreating the same logic of the internal data resolver handling.

> The second seems simpler, and simultaneously worrying in how it's "recreating" an existing behavior. However, I think it may not be too much an issue in practice, both because (a) the logic is quite simple and reuses common resolution selectors, and (b) it could be abstracted rather easily to a higher-order function common to both of these new resolvers.

Implemented in ccf5ba0.